src/components/CallView/CallView.vue

```diff
@@ -196,11 +201,16 @@ export default {
 		const screenshotMode = ref(false)
 		provide('CallView:screenshotModeEnabled', screenshotMode)
 
+		const isBackgroundBlurred = ref(serverSupportsBackgroundBlurred
+			? null
+			: BrowserStorage.getItem('background-blurred') !== 'false')
```

Comment:
```suggestion
		const isBackgroundBlurred = ref(BrowserStorage.getItem('background-blurred') !== 'false')
```

Because you would need to evaluate its value only when `serverSupportsBackgroundBlurred` is falsy anyway